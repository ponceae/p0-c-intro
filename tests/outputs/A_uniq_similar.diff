--- outputs/A_uniq_similar.txt	2021-09-17 16:13:18.820471006 -0400
+++ expected/A_uniq_similar.txt	2021-08-24 14:13:54.000000000 -0400
@@ -1 +1,8 @@
-Invalid argument.
+aaa
+abb
+aab
+bbb
+bba
+baa
+bbb
+ccc
