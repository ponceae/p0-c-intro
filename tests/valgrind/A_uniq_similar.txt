==3257532== Memcheck, a memory error detector
==3257532== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3257532== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3257532== Command: ../intro -u inputs/A_uniq_similar.txt
==3257532== 
../intro: invalid option -- 'u'
Invalid argument.
==3257532== 
==3257532== HEAP SUMMARY:
==3257532==     in use at exit: 0 bytes in 0 blocks
==3257532==   total heap usage: 1 allocs, 1 frees, 8,192 bytes allocated
==3257532== 
==3257532== All heap blocks were freed -- no leaks are possible
==3257532== 
==3257532== For lists of detected and suppressed errors, rerun with: -s
==3257532== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
