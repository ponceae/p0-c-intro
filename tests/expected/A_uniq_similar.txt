aaa
abb
aab
bbb
bba
baa
bbb
ccc
